### ISSUE TITLE
Python system tests on Windows do not generate code coverage data

### ISSUE THREAD

**Issue by andrewkroh** [elastic/beats#599]
The process under test is not shutdown gracefully through a signal handler due to the way [Popen.terminate()](https://docs.python.org/2/library/subprocess.html#subprocess.Popen.terminate) works on Windows. This causes the coverage report (`.cov`) file to not be written to disk.

**Comment by tsg** [comment]
@andrewkroh is this valid for all beats or only a particular one?

**Comment by andrewkroh** [comment]
I just surveyed the `system-test-outputs.zip` [artifacts](https://ci.appveyor.com/project/elastic-beats/beats/build/501/job/gu0n9me7x4sye0pc/artifacts) from each of the AppVeyor builds. Topbeat and Filebeat are missing `.cov` files. Winlogbeat has them.

I made a [change](https://github.com/elastic/beats/commit/ee446a4036aa97b96147d45225655dfb850fef3b#diff-714b8651f2c293a867918ab7def84ec0L25) to winlogbeat.py to see if I could get it working and it appears to have worked. So I think we should apply a this workaround to each beat. Agree?
